Authentication/Login/View/LoginConfiguration.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 */
 public protocol LoginControllerTransitionDelegate {
```

Comment:
When the user logs in, the bootstrapper automatically opens your app's main view. Do you think someone may want to do something before that?
@guidomb